client-v2/src/shared/actions/__tests__/snapcards.spec.ts

```diff
@@ -80,19 +83,27 @@ describe('Snap cards actions', () => {
         '/http/proxy/https://www.theguardian.com/example/tag/page?view=mobile',
         guardianTagPage
       );
-      (window as any).confirm = jest.fn(() => true);
-      const store = mockStore(initialState);
-      await store.dispatch(createArticleEntitiesFromDrop(
+      const store = configureStore(initialState);
+      const promise = store.dispatch(createArticleEntitiesFromDrop(
         idDrop('https://www.theguardian.com/example/tag/page')
       ) as any);
-      const actions = store.getActions();
-      expect(actions[0]).toEqual(
-        cardsReceived({
-          card1: await createLatestSnap(
-            'https://www.theguardian.com/example/tag/page',
-            'Example title'
-          )
-        })
+      // We can't wait for the entire promise to be done here -- we need to call the modal
```

Comment:
Yep, for the record, this works until we have an effective way to mock modal acceptance, although it's not ideal